docs/developer/community/faq.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: Frequently Asked Questions
+sidebar_label: FAQ
+---
+
+## I cannot log in to the dashboard. I receive an error: ‘Saleor ran into an unexpected problem’. What actions should I take?
+
+Make sure that you have set the environment variables.
+
+There are two environment variables available for configuration:
+
+- `API_URI` (required): URI of the GraphQL API endpoint of a running instance of Saleor. If you are running Saleor locally with the default settings, set `API_URI` to: `http://localhost:8000/graphql/`.
+
+- `APP_MOUNT_URI`: URI at which the Dashboard app will be mounted. E.g. if you set `APP_MOUNT_URI` to `/dashboard/`, your app will be mounted at `http://localhost:9000/dashboard/`.
+
+- `STATIC_URL`: URL where the static files are located. E.g. if you use S3 bucket, you should set it to the bucket's URL. By default, Saleor assumes you serve static files from the root of your site at `http://localhost:9000/`.
+
+You can find the instructions on how to install and run the dashboard here:
+
+https://github.com/mirumee/saleor-dashboard#getting-started
+
+If you still cannot log in to the dashboard, please contact us on Spectrum.
+
+## How to make Saleor work in production with Ubuntu 19.10/Nginx/Gunicorn server?
+
+You will need to perform the following actions:
+
+- Build the storefront and the dashboard.
+
+- Upload both to separate folders on the server.
+
+- Create separate location entries in your Nginx config for the backend, the dashboard, and the storefront.
+
+    ```
+    location /graphql/ {
+        proxy_pass http://localhost:8000;
+    }
+
+    location / {
+        root /path/to/your/storefront;
+    }
+
+    location /dashboard/ {
+        root /path/to/your/dashboard;
+    }
```

Comment:
I don't think it works, should be `alias` here as it would look for `dashboard/index.js` for example rather than `index.js`

Reply:
I could be wrong though